packages/fluentui/react-icons-northstar/src/components/DoorArrowLeftIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { createSvgIcon } from '../utils/createSvgIcon';
+import { iconClassNames } from '../utils/iconClassNames';
+
+export const DoorArrowLeftIcon = createSvgIcon({
+  svg: ({ classes }) => (
+    <svg
+      style={{ overflow: 'visible' }}
+      role="presentation"
+      focusable="false"
+      viewBox="2 2 16 16"
```

Comment:
This one is cut off... I can't remember, is it the styling that allows it to overflow the container?

Reply:
Yeah, that overflow: visible should fix the issue

Reply:
it doesn't appear that way because i was lazy and didn't change the style attribute to style='overflow:visible' in codepen